server/services/grid_service.py

```diff
@@ -57,3 +73,71 @@ def _update_feature(clip_to_aoi: bool, feature: dict, new_shape):
             feature['properties']['splittable'] = False
         return feature
 
+    @staticmethod
+    def _parse_geojson(input: str)->list:
```

Comment:
Suggest renaming this to:

_to_shapely_features(geojson_collection: str) -> list